src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+averaged strain. The LCS operator coupled with the Integral Curve (Hyperbolic
+LCS) and Limit Cycle (Elliptical LCS) operators provide a mechanism to
+calculate these structures and have been developed in cooperation between the
+VisIt PICS team and the Haller Group.
+
+INCLUDE IMAGE of elliptical and hyperbolic LCS
+
+For complete details on computing elliptical LCS (closed orbits) and hyperbolic
+LCS (stretch and shrink lines) see the following paper: `K. Onu, F. Huhn, & G.
+Haller, LCS Tool: A Computational platform for Lagrangian coherent structures,
+J. of Computational Science, 7 (2015) 26-36
+<http://georgehaller.com/reprints/lcstool.pdf>`_
+
+.. toctree::
+    :maxdepth: 1
+
+    IntegralCurve_operator
+    LCS_operator
+    LimitCycle_operator
+    Poincare_operator
+
+Parameters
+----------
+
+Common to all PICS operators is a four tab GUI: Source, Integration,
+Appearance, and Advanced (the Poincare operator also has an Analysis tab).
+These tabs contain many functions that are common across all four operators.
+The following is a description of those common features. Please note: on a mac
+system, the contents of the Source and Integration tabs are combined into a
+single Integration tab.
+
+
+Source
+^^^^^^
+
+The set of points to seed the integral curves in the operator. See specific
+operator for details.
+
+Field
+"""""
+
+Sets the field type so that the native elements are used when interpolating the
+vector fields. Each operator provides the following options:
+
+Default
+    Use VisIt's native VTK mesh structure performing linear interpolation on
+    the vector field.
+    
+Flash
+    Evaluates the velocity field via the Lorentz force. Parameters are:
+
+    * Constant - A constant multiple applied to the velocity.
+    * Velocity - When combined with Leap-Frog integration, this sets the
+    initial velocity used in the integration.
+   
+M3D-C1 2D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial.
+    Parameters are:
+
+    * Constant - A constant multiple applied to the perturbed part of the
+      field.
+   
+M3D-C1 3D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial and
+    1D toroidal 4th order Bezier spline.
+    
+Nek5000
+    Evaluates the 3D vector field using `Nek5000 <https://nek5000.mcs.anl.gov/>`_
+    spectral elements.
+    
+Nektar++
+    Evaluates the 3D vector field using `Nektar++ <https://www.nektar.info/>`_
+    spectral elements.
+        
+NIMROD
+    Evaluates the 3D magnetic field via a 2D poloidal 3rd order Lagrange-type
+    element and 1D toroidal finite Fourier series element (future
+    implementation).
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for the numerical integrator. See each operator for varied
+settings.
+
+
+Integrator
+""""""""""
+
+Sets the integration scheme. Options are:
+
+Forward Euler
+    The basic Euler integration with a fixed step length. This integrator is
+    not recommended.
```

Comment:
If it is not recommended, why is it here? What *does* it do that suggests it still needs to exist in the interface. Maybe a question for @ARSanderson

Reply:
I don't think it is still needed here. Forward Euler is the "tribute" that all integration packages pay. Because it is the very first numerical scheme we learn, we like to include it. Its successors greatly improve upon what it does, but we still pay tribute. This is especially true if our software is used for education.

All that said, I think it is safe to remove. There is nothing that Forward Euler can do that the other integrators here cannot do. I'll make a note to remove it.

Reply:
People often use it because it is quick and dirty and used as comparison. I just worked with Nektar++ and most all integration schemes reduce down to either forward or backwards Euler. I would just leave it in.